### ISSUE TITLE
Confconsole Dehydrated plugin appears broken on v14.2

### ISSUE THREAD

**Issue by JedMeister** [turnkeylinux/tracker#976]
[Dan has reported in the forums](https://www.turnkeylinux.org/forum/support/fri-20171124-1958/dehydrated-out-date) that Confconsole is not working and is throwing an error:
```
2017-11-22 18:22:32] dehydrated-wrapper: WARNING: /etc/dehydrated/confconsole.config not found; copying default from /usr/share/confconsole/letsencrypt/dehydrated-confconsole.config
[2017-11-22 18:22:32] dehydrated-wrapper: WARNING: /etc/dehydrated/confconsole.hook.sh not found; copying default from /usr/share/confconsole/letsencrypt/dehydrated-confconsole.hook.sh
[2017-11-22 18:22:32] dehydrated-wrapper: WARNING: /etc/cron.daily/confconsole-dehydrated not found; copying default from /usr/share/confconsole/letsencrypt/dehydrated-confconsole.cron
  + ERROR: An error occurred while sending post-request to https://acme-v01.api.letsencrypt.org/acme/new-reg (Status 400)
Details:
{
  "type": "urn:acme:error:malformed",
  "detail": "Provided agreement URL [https://letsencrypt.org/documents/LE-SA-v1.1.1-August-1-2016.pdf] urn:acme:error:malformed [https://letsencrypt.org/documents/LE-SA-v1.2-November-15-2017.pdf]",
  "status": 400
}
[2017-11-22 18:22:35] dehydrated-wrapper: FATAL: dehydrated exited with a non-zero exit code.
[2017-11-22 18:22:35] dehydrated-wrapper: WARNING: Something went wrong, restoring original cert & key.
[2017-11-22 18:22:36] dehydrated-wrapper: WARNING: Check today's previous log entries for details of error.
```

**Comment by JedMeister** [comment]
I have just confirmed this and am working on a fix (or at least a workaround).

**Comment by JedMeister** [comment]
The workaround is really simple! You just need to add a new line to the current conf file. You can do that like this:
```
echo 'LICENSE="https://letsencrypt.org/documents/LE-SA-v1.2-November-15-2017.pdf"' >> /etc/dehydrated/confconsole.config
```

**Comment by JedMeister** [comment]
This has already been fixed for the upcoming v15.0 release, but is still an issue for v14.2. Hopefully it will get uploaded soon!

**Comment by JedMeister** [comment]
Whilst ideally this should also be updated in v14.2, due to our current focus on pushing forward with v15.0, I'm going to close this issue... Apologies to any remaining v14.2 users, but the [above workaround](https://github.com/turnkeylinux/tracker/issues/976#issuecomment-347433665) is still relevant and will allow you to continue on. However, please note that in the v15.0 version, it dynamically gets the URL of the agreement, so should be somewhat "future proof"...
